site/src/utilities.ts

```diff
@@ -259,7 +259,7 @@ function getPrerequisiteFormatHtml(
       .map((childPrereq) =>
         getPrerequisiteFormatHtml(priorCourses, childPrereq, false)
       )
-      .join(` ${prereq.type} `);
+      .join(prereq.type);
```

Comment:
Why was this changed?

Reply:
Yes, it was a mistake that I didn't notice when I checked. It's been undone in the latest commits.